portfolio/src/main/webapp/script.js

```diff
@@ -40,8 +41,10 @@ function getComments() {
             getElementById("comment-container");
       commentListElement.innerHTML = '';
       comment.forEach((element) => {
+        console.log(element.imageUrl);
         commentListElement.appendChild(createCommentElement(
```

Comment:
This is where I would check for a valid imageUrl before creating an image from the comment element. Either here or in createCommentElement (assuming that's where you are going to add the image).